content/en/continuous_integration/setup_tests/dotnet.md

```diff
@@ -230,6 +230,16 @@ To use the custom instrumentation in your .NET application:
 
 For more information about how to add spans and tags for custom instrumentation, see the [.NET Custom Instrumentation documentation][8].
 
+## Information collected
+
+When CI Visibility is enabled, the following data is collected from your project:
+
+* Test names and durations.
+* Predefined environment variables set by CI providers.
+* Git commit history including the hash, message, author information, and files changed (without file contents).
+* Code coverage information, including file names and line numbers covered by each test.
```

Comment:
This is only true in case ITR is enabled - maybe we have two lists, one for CI Visibility enabled, and another one for CI Visibility + ITR enabled?

Reply:
👍 I've split that into a separate list for languages where we have ITR support.